src/Parser/FrostMage/Modules/Features/CooldownTracker.js

```diff
@@ -0,0 +1,133 @@
+import CoreCooldownTracker, { BUILT_IN_SUMMARY_TYPES } from 'Parser/Core/Modules/CooldownTracker';
+
+import SPELLS from 'common/SPELLS';
+
+const debug = false;
+
+class CooldownTracker extends CoreCooldownTracker {
+  static cooldownSpells = [
+    ...CooldownTracker.cooldownSpells,
+    {
+      spell: SPELLS.ICY_VEINS,
+      summary: [
+        BUILT_IN_SUMMARY_TYPES.DAMAGE,
+      ],
+    },
+  ];
+
+  static castCooldowns = [
+    {
+      spell: SPELLS.FROZEN_ORB_CAST,
+      duration: 15,
+      summary: [
+        BUILT_IN_SUMMARY_TYPES.DAMAGE,
+      ],
+    },
+    {
+      spell: SPELLS.RUNE_OF_POWER,
+      duration: 10,
+      summary: [
+        BUILT_IN_SUMMARY_TYPES.DAMAGE,
+      ],
+    },
+    {
+      spell: SPELLS.MIRROR_IMAGE,
+      duration: 40,
+      summary: [
+        BUILT_IN_SUMMARY_TYPES.DAMAGE,
+      ],
+    },
+    {
+      spell: SPELLS.HEROISM,
+      duration: 40,
+      summary: [
+        BUILT_IN_SUMMARY_TYPES.DAMAGE,
+      ],
+    },
+    {
+      spell: SPELLS.BLOODLUST,
+      duration: 40,
+      summary: [
+        BUILT_IN_SUMMARY_TYPES.DAMAGE,
+      ],
+    },
+    {
+      spell: SPELLS.TIME_WARP,
+      duration: 40,
+      summary: [
+        BUILT_IN_SUMMARY_TYPES.DAMAGE,
+      ],
+    },
+    {
+      spell: SPELLS.NETHERWINDS,
+      duration: 40,
+      summary: [
+        BUILT_IN_SUMMARY_TYPES.DAMAGE,
+      ],
+    },
+    {
+      spell: SPELLS.ANCIENT_HYSTERIA,
+      duration: 40,
+      summary: [
+        BUILT_IN_SUMMARY_TYPES.DAMAGE,
+      ],
+    },
+    {
+      spell: SPELLS.DRUMS_OF_FURY,
+      duration: 40,
+      summary: [
+        BUILT_IN_SUMMARY_TYPES.DAMAGE,
+      ],
+    },
+    {
+      spell: SPELLS.DRUMS_OF_RAGE,
+      duration: 40,
+      summary: [
+        BUILT_IN_SUMMARY_TYPES.DAMAGE,
+      ],
+    },
+    {
+      spell: SPELLS.DRUMS_OF_THE_MOUNTAIN,
+      duration: 40,
+      summary: [
+        BUILT_IN_SUMMARY_TYPES.DAMAGE,
+      ],
+    },
```

Comment:
Bloodlust-type effects shouldn't be here. `castCooldowns` is only for cooldowns that doesn't place a buff on the player.

Reply:
Got it, ill get that fixed.

Reply:
So just to make sure i got this right, if it puts a buff on me it should be in cooldownSpells (like Icy Veins and Rune of Power), and if it doesnt put a buff on me it goes in castCooldowns (like Mirror Image and Frozen Orb)?

Would Bloodlust effects therefore go in cooldownSpells? or should it be removed alltogether?